js/src/common/components/Page.js

```diff
@@ -40,6 +40,9 @@ export default class Page extends Component {
     if (this.bodyClass) {
       $('#app').addClass(this.bodyClass);
     }
+
+    // hotfix the redraw of header components for beta.14
```

Comment:
Is this meant as a temporary or permanent fix? If temporary, where could we improve this? If permanent, the comment should be revised imo.

Reply:
The new PR is probably a better solution: https://github.com/flarum/core/pull/2390